client/rest/src/test/java/org/opensearch/client/RestClientBuilderIntegTests.java

```diff
@@ -68,6 +71,16 @@ public class RestClientBuilderIntegTests extends RestClientTestCase {
 
     private static HttpsServer httpsServer;
 
+    static {
+        int highestPriority = 1;
+        if (Security.getProvider(BouncyCastleFipsProvider.PROVIDER_NAME) == null) {
+            Security.insertProviderAt(new BouncyCastleFipsProvider(), highestPriority++);
+        }
+        if (Security.getProvider(BouncyCastleJsseProvider.PROVIDER_NAME) == null) {
+            Security.insertProviderAt(new BouncyCastleJsseProvider(), highestPriority);
```

Comment:
Why do we need 2 providers here? I would assume `BouncyCastleFipsProvider` - in FIPS mode, oherwise `BouncyCastleJsseProvider`?